pytri/__init__.py

```diff
@@ -515,4 +520,6 @@ def mesh(self, *args, **kwargs):
         color = kwargs.get("color", "#00bbee")
 
         mesh = Mesh(geometry=geo, material=MeshLambertMaterial(color=color))
-        return self._add_layer(mesh)
+        ret = self._add_layer(mesh)
+        self._reposition_camera_on_bbs()
```

Comment:
I don't love the idea of repositioning the camera without the user asking for it. I was thinking:

```python
from pytri import Figure

fig = Figure(auto_camera=True)
fig.blah()
```

or,

```python
from pytri import Figure

fig = Figure()
fig.blah()
fig.recenter_camera()
```
?